src/app/Page/main-view/main-view.component.html

```diff
@@ -2,75 +2,30 @@
   <!--* board bar and columns HTML -->
 
   <div class="board">
-    <div class="baord-wrapper">
+    <div class="baord-wrapper" cdkDropListGroup> 
 
-        <div class="board-columns" cdkDropListGroup>
+        <div class="board-columns" *ngFor="let employeeGroup of employees">
             <div class="board-column">
                 <div class="column-title">
-                    Employees
+                    {{employeeGroup.description}}
                 </div>
                 <div class="task-container"
                 cdkDropList
                 #todoList="cdkDropList"
                 (cdkDropListDropped)="drop($event)"
-                cdkDropListData="0"
-                >
+                [cdkDropListData]="employeeGroup" >
```

Comment:
passing the new type down, this has both the ID and the collections we need to pass into your existing code.